UPGRADE-2.0.md

```diff
@@ -122,6 +122,18 @@
     * `Sylius\Bundle\AttributeBundle\Doctrine\ORM\AttributeValueRepository`
       extends `Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository`
       implements `Sylius\Component\Attribute\Repository\AttributeValueRepositoryInterface`
+  
+  Product:
+
+    * `Sylius\Bundle\ProductBundle\Doctrine\ORM\ProductAssociationRepository`
+    * `Sylius\Bundle\ProductBundle\Doctrine\ORM\ProductAssociationTypeTranslationRepository`
+    * `Sylius\Bundle\ProductBundle\Doctrine\ORM\ProductAttributeRepository`
+    * `Sylius\Bundle\ProductBundle\Doctrine\ORM\ProductAttributeTranslationRepository`
+    * `Sylius\Bundle\ProductBundle\Doctrine\ORM\ProductOptionTranslationRepository`
+    * `Sylius\Bundle\ProductBundle\Doctrine\ORM\ProductOptionValueRepository`
+    * `Sylius\Bundle\ProductBundle\Doctrine\ORM\ProductOptionValueTranslationRepository`
+    * `Sylius\Bundle\ProductBundle\Doctrine\ORM\ProductTranslationRepository`
+    * `Sylius\Bundle\ProductBundle\Doctrine\ORM\ProductVariantTranslationRepository`
```

Comment:
```suggestion
    * `Sylius\Bundle\ProductBundle\Doctrine\ORM\ProductVariantTranslationRepository` has been added
```

Reply:
![CleanShot 2023-11-20 at 16 40 17@2x](https://github.com/Sylius/Sylius/assets/80641364/bdc43f68-ad3d-4732-b2bd-98722a5acccb)
IMO the `has been added` is pointless here, if previously it was included, we can remove it instead adding it here.